### PR TITLE
feat(front-end): Added basic views for actually viewing visits.

### PR DESCRIPTION
**Pull request by nchapman**
Visualized visits are here!

screen shot 2015-01-07 at 3 15 02 pm

You'll need to have run `server/db/create_test_data.js` to see anything at all.

@pdehaan @6a68 r?

### CODE REVIEW

File: app/scripts/views/visits/index.js [hunk 1]
```diff
@@ -4,12 +4,27 @@
...
+], function (BaseView, VisitsIndexTemplate, Visits, VisitsItemView) {
   'use strict';
 
   var VisitsIndexView = BaseView.extend({
     template: VisitsIndexTemplate,
+
+    initialize: function () {
+      this.collection = new Visits();
+
+      this.listenTo(this.collection, 'add destroy reset', this.render);
```
Comment (nchapman) [inline app/scripts/views/visits/index.js]: Hmm it might be a bad idea to trigger render on reset and add. I'll look into this.

Comment (nchapman) [inline app/scripts/views/visits/index.js]: Seems to be a-ok; leaving it as is.

File: app/scripts/views/visits/item.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+
+    initialize: function () {
+      this.listenTo(this.model, 'change', this.render);
+    },
+
+    destroyModel: function (event) {
+      event.preventDefault();
+
+      if (window.confirm('Destroy this visit?')) {
```
Comment (pdehaan) [inline app/scripts/views/visits/item.js]: A prettier UX, i have not seen!

Comment (nchapman) [inline app/scripts/views/visits/item.js]: You know it!

**Comment by 6a68** [comment]
awww yissssss

**Pull request merged**